src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas2/ApplicationNoteService.kt

```diff
@@ -48,13 +53,32 @@ class ApplicationNoteService(
 
     val savedNote = saveNote(application, note.note, user)
 
+    if (isExternalUser) {
```

Comment:
Is this something I'd need to check on my branch too (emailing the referrer when the status has updated on an application)?

Reply:
hmm I don't think so currently, because only an assessor has permission to update the status, so `isExternalUser` should always be true